content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -3,399 +3,1025 @@ title: Using Windows Server Containers in Kubernetes
+The following networking functionality is not supported on Windows nodes
 
-#### Host-Gateway Topology
-This topology is similar to the Upstream L3 Routing topology with the only difference being that static IP routes are configured directly on each cluster node and not in the upstream ToR. Each node uses a local 'l2bridge' network with a pod CIDR assigned as before and has routing table entries for all other pod CIDR subnets assigned to the remote cluster nodes.
+* Host networking mode is not available for Windows pods
+* Local NodePort access from the node itself will fail (works for other nodes or external clients)
+* Accessing service VIPs from nodes will be available with a future release of Windows Server 
+* Overlay networking support in kube-proxy is an alpha release. In addition, it requires [KB4482887](https://support.microsoft.com/en-us/help/4482887/windows-10-update-kb4482887) to be installed on Windows Server 2019
+* Outbound communication using the ICMP protocol via the win-overlay, win-bridge, and Azure-CNI plugin. Specifically, the Windows data plane ([VFP](https://www.microsoft.com/en-us/research/project/azure-virtual-filtering-platform/)) doesn't support ICMP packet transpositions. This means:
+  * ICMP packets directed to destinations within the same network (e.g. pod to pod communication via ping) will work as expected and without any limitations
+  * TCP/UDP packets will work as expected and without any limitations
+  * ICMP packets directed to pass through a remote network (e.g. pod to external internet communication via ping) cannot be transposed and thus will not be routed back to their source
+  * Since TCP/UDP packets can still be transposed, one can substitute **ping <destination>** with **curl <destination>** to be able to debug connectivity to the outside world.
 
-#### Using OVN with OVS
-The following diagram gives a general overview of the architecture and interaction between components:
+##### CNI Plugins
 
-![Overlay using OVN controller and OVS Switch Extension](ovn_kubernetes.png)
+* Windows reference network plugins win-bridge and win-overlay do not currently implement [CNI spec](https://github.com/containernetworking/cni/blob/master/SPEC.md) v0.4.0 due to missing "CHECK" implementation.
+* The Flannel VXLAN CNI has the following limitations on Windows:
 
-(The above image is from [https://github.com/openvswitch/ovn-kubernetes#overlay-mode-architecture-diagram](https://github.com/openvswitch/ovn-kubernetes#overlay-mode-architecture-diagram))
+1. Node-pod connectivity isn't possible by design. It's only possible for local pods with Flannel PR[ https://github.com/coreos/flannel/pull/1096](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcoreos%2Fflannel%2Fpull%2F1096&data=02%7C01%7CDavid.Schott%40microsoft.com%7Cfa26c088fac743188e7d08d69ea3d836%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636870823885740325&sdata=1QEvw9Gh6IioT7ruVhhXAlSgN1a%2FMsqn4ViDQSSSVjs%3D&reserved=0)
```

Comment:
There appears to be some formatting error here, can we update the URL to [PR1097](https://github.com/coreos/flannel/pull/1096)

Reply:
fixed